test/e2e/utils/kubernetes.go

```diff
@@ -210,7 +210,7 @@ func (k TestKubernetes) PrintAllResources(namespace string, list runtime.Object,
 	unstructuredResourceList.SetUnstructuredContent(unstructuredResource)
 
 	for _, item := range unstructuredResourceList.Items {
-		yaml, err := yaml.Marshal(item)
+		yaml_, err := yaml.Marshal(item)
```

Comment:
Any reason for the var name change?

Reply:
Just changed the name of that var internally to avoid any collision with the imported package name... I thought it was a nit, but if you prefer we may keep its previous name.

Reply:
No that's fine, makes sense :+1: